src/trompequihash/blake/blake2-config.h

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Given the changes @str4d described, I think there is no longer any need to include a BLAKE2b implementation separate from the one we use from libsodium, so strip out all of that code.